### ISSUE TITLE
NTR: Cytoplasmic inclusions or granules in fibroblasts (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9106]
New term request Cytoplasmic inclusions or granules in fibroblasts (segal_080118203638): I-cells included  UMLS: 1854889

**Comment by pnrobinson** [comment]
@MickeySegal
this term is both broad (any kind of inclusion or granule) and specific (fibroblasts), but I am wondering if it is really intended to be specific to fibroblasts (e.g., inclusions in fibroblasts but not in other cells). 
Currently, HPO has a number of specific terms such as Neutrophil inclusion bodies -- I am thinking that either we should make a common parent term (Inclusion bodies) that should be used for inclusion bodies where we do not really know what cell is important, or maybe it is justified to have a special fibroblast term as well?
Thoughts?

**Comment by pnrobinson** [comment]
@MickeySegal
Is this a specifically fibroblast phenotype or is it just that we happened to measure in fibroblasts for convenience?
Currently, HPO has a bunch of terms like X inclusion bodies, e.e.,g Neutrophil inclusion bodies HP:0008264, that are not connected. I am thinking of making a term "Inclusion bodies" under Abnormal cellular phenotype HP:0025354 -- let me know if there is a reason to add Fibroblast to the list of cells.

**Comment by MickeySegal** [comment]
Our finding is based on the following 2 mucolipidosis articles:

https://www.ncbi.nlm.nih.gov/books/NBK1828/
The finding by Jules Leroy and Robert DeMars by phase-contrast microscopy of large amounts of dark and dense granules filling almost the entire cytoplasm of cultured fibroblasts from skin biopsies of two unrelated children resulted in the use of the term "inclusion cells," abbreviated as "I-cells." Hence, the term "I-cell disease" was introduced.

https://www.ncbi.nlm.nih.gov/books/NBK1875/
The cytoplasm is filled with dense granular inclusions that consistently spare a juxtanuclear zone that represents the endoplasmic reticulum and the Golgi apparatus. Electron microscopic study shows that the inclusions are swollen lysosomes bound by a unit membrane and filled with heterogeneous material of varying texture, shape, and electron density. The fibroblasts were originally called inclusion cells (I-cells) and the disorder associated with this in vitro [phenotype](https://www.ncbi.nlm.nih.gov/books/n/gene/glossary/def-item/phenotype/) "I-cell disease."

If you have a choice of terminology that describes this and fits better with your ontology, we'd be glad to use that.

**Comment by pnrobinson** [comment]
For mucolipidosis, the I cells are not limited to cultured fibroblasts
https://www.nature.com/articles/s41390-018-0234-2

**Comment by pnrobinson** [comment]
@MickeySegal 
the main finding in Mucolipidosis is Vacuolated lymphocytes HP:0001922 
The HPO will need to work on the inclusion body hierarchy in the future, but for now I have added this

Adding new term: 
- ID: HP:6001025
- label: Pathological inclusion body
- parents: [Abnormal cell morphology (HP:0025461)](https://hpo.jax.org/app/browse/term/HP:0025461)
- pmid: 
- def: Pathological inclusion bodies are are composed of glycogen, proteins, nucleic acids, lipids, hemosiderin, and calcium, that accumulate as metabolic by-products, catabolic products of macromolecular complexes, or in the context of cell injury.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9106
- synonyms:

**Comment by MickeySegal** [comment]
The B-cell tests sounds like an improvement because it is minimally invasive.
